docs/source/federation-spec.md

```diff
@@ -413,3 +413,41 @@ type User @key(fields: "id") {
   name: String @override(from: "SubgraphA")
 }
 ```
+
+### `@tag`
+
+```graphql
+directive @tag(from: String!) on FIELD_DEFINITION | INTERFACE | OBJECT | UNION | ARGUMENT_DEFINITION | SCALAR | ENUM | ENUM_VALUE | INPUT_OBJECT | INPUT_FIELD_DEFINITION
+```
+
+The `@tag` directive is used to provide a mechanism for applying arbitrary string metadata to the fields and types of a schema. This metadata is potentially useful throughout the schema’s lifecycle, including, but not limited to, processing, static analysis, and documentation.
+
+```graphql
+schema
+  @core(feature: "https://specs.apollo.dev/core/v0.2")
```

Comment:
should this be `@link`ed instead? i.e. nowhere else in the docs we mention `@core` directive in our examples

Reply:
Yes, should be @link. @core was the old way. Also, we should get the @tag from the federation CoreFeature.

```@link(url: "https://specs.apollo.dev/federation/v2.0", import: ["@tag"])```